.travis.yml

```diff
@@ -37,14 +38,28 @@ install:
   - if [[ $SETUP == 'full' ]]; then conda create --yes -q -n pyenv python=$PYTHON_VERSION numpy scipy nose=1.3.7 sphinx=1.3 griddataformats six; fi
   - if [[ $SETUP == 'minimal' ]]; then conda create --yes -q -n pyenv python=$PYTHON_VERSION numpy nose=1.3.7 sphinx=1.3 griddataformats six; fi
   - source activate pyenv
-  - if [[ $SETUP == 'full' ]]; then conda install --yes cython biopython matplotlib networkx netcdf4; fi
+  - |
+    if [[ $SETUP == 'full' ]]; then \
+        conda install --yes cython biopython matplotlib networkx netcdf4; \
+        conda install -c biobuilds --yes clustalw=2.1; \
+    fi
   - if [[ $SETUP == 'minimal' ]]; then conda install --yes cython biopython networkx; fi
 # ensure that cython files are rebuilt
   - find . -name '*.pyx' -exec touch '{}' \;
   - pip install -v package/
   - pip install testsuite/
   - pip install coveralls
   - chmod +x testsuite/MDAnalysisTests/mda_nosetests
+# additional external tools (Issue #898) -- HOLE
+  - |
+     if [[ $SETUP == 'full' ]]; then \
+         bash ./maintainer/install_hole.sh $TRAVIS_OS_NAME "${HOME}/${MDA_OPTPACKAGES}"; \
+         HOLE_BINDIR="${HOME}/${MDA_OPTPACKAGES}/hole2/exe"; \
+         export PATH=${PATH}:${HOLE_BINDIR}; \
+     fi
+  # we should still be here but make doubly sure:
+  - cd ${TRAVIS_BUILD_DIR}
```

Comment:
I tried to make it a bit future-proof: You can put other installation scripts into this `if` statement. But if you are jumping directories then thr rest of the travis script will fail. The "making doubly sure" serves as a reminder that one of the assumptions of the travis script is that everything happens from within the checked-out git repo.